lint/rule_validate.go

```diff
@@ -0,0 +1,105 @@
+package lint
+
+import (
+	"sync"
+
+	"cuelang.org/go/cue/errors"
+	"github.com/grafana/grafana/pkg/schema"
+	"github.com/grafana/grafana/pkg/schema/load"
```

Comment:
Are these packages apache licensed? This tool can't import AGPL code (its imported by mixtool, which is apache licensed).

Reply:
+1 this concern. Seems like the schema bits of grafana could/should/will-eventually be under a more forgiving license?

Reply:
Hmm, looks like we need to update the list. All the CUE-related schema stuff is supposed to be Apache 2 (which is why https://github.com/grafana/grafana/blob/main/LICENSING.md excludes `cue`), but looks like i didn't add a line for this subtree in particular when this package was added. Will make a PR, for this and a couple others.

(pkg/schema will become grafana/scuemata, which [is A2](https://github.com/grafana/scuemata/blob/main/LICENSE))